static/src/stylesheets/module/content-garnett/_content.global.scss

```diff
@@ -12,6 +12,15 @@
 }
 
 .meta__numbers {
+    @include mq($from: leftCol, $until: wide) {
+        // Excude templates that have a full width meta
+        .content:not(.content--liveblog):not(.content--gallery):not(.content--interactive) .meta__extras:not(.meta__extras--crossword) & {
```

Comment:
beyond the scope of this i know, so no problem with it in this PR, but we coudl really do with removing the need for these selectors, they're really brittle... 😰

Reply:
Yep definitely! Adding to my list.